.github/workflows/default.yml

```diff
@@ -49,8 +49,16 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        gradle: [ 7.0.1, current, rc ]
-        java: [ 8, 11, 16 ]
+        gradle: [ 7.3, current, rc ]
+        java: [ 8, 11, 17 ]
```

Comment:
I thought it makes sense to test on Java 17. I expect more people start using it as it is an LTS version

Reply:
Agree